codalab/lib/formatting.py

```diff
@@ -23,16 +22,7 @@ def contents_str(input_string, verbose=False):
     """
     if input_string is None:
         return NONE_PLACEHOLDER if verbose else ''
-
-    # Unicode is always printable
-    if isinstance(input_string, str):
-        return input_string
-
-    # Assume string is UTF-8 or else it contains arbitrary binary data that should not be rendered
-    try:
-        return input_string.decode('utf-8')
-    except UnicodeDecodeError:
-        return BINARY_PLACEHOLDER if verbose else ''
+    return str(input_string)
```

Comment:
Why did we return BINARY_PLACEHOLDER for non-utf8-binary files before? Was this the previous behavior in Python 2, or something that got introduced in the Python 3 transition?

Reply:
Before, this function could take bytes, but now we assume it can't.